doc/superbuild/source/developers/releases/Checklist.rst

```diff
@@ -65,50 +65,69 @@ This is a list of tasks that need to be done before a SUNDIALS release.
 The order is bottom-up starting with solver source files and ending with
 web pages.
 
-#. Create release branch from trunk in the Git repository
+#. Create release branch ``release/vX.Y.Z`` from develop in the git repo.
+
+#. Update version numbers and release date information using the ``updateVerson.sh``
+   script.
```

Comment:
This has to be run from the `scripts` directory (for now, I have a branch that will remove this requirement)
```suggestion
   script.
   
   .. code-block:: shell
      pushd scripts/ && ./updateVersion.sh && popd
```